files/image_config/misc/docker-wait-any

```diff
@@ -11,13 +11,12 @@
     stopped running. Here, we spawn multiple threads and wait on one
     container per thread. If any of the threads exit, the entire
     application will exit.
-    NOTE: This script is written against docker-py version 1.6.0. Newer
-    versions of docker-py have a different API.
+    NOTE: This script is written against docker 4.1.0.
```

Comment:
@jleveque to check.

Reply:
According to API docs, this looks good. However, I suggest adding the rest of my note back in the comment, as follows:

```
    NOTE: This script is written against 'docker' Python package version 4.1.0.
    Newer versions of docker may have a different API.
    
````